CMakeLists.txt

```diff
@@ -136,7 +136,6 @@ configure_package_config_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake/eudaqConfig.cmak
                               INSTALL_DESTINATION cmake
                               PATH_VARS EUDAQ_INCLUDE_DIRS EUDAQ_LIBRARIES_DIRS
                               )
-
```

Comment:
Why this change in the general top level CMakeLists.txt?

Reply:
Sorry about that. I rolled it back.